tests/periph_gpio_arduino/README.md

```diff
@@ -0,0 +1,71 @@
+# Arduino peripheral GPIO Test
+
+This test is designed for testing the Arduino to RIOT pin mapping
+on boards with Arduino support.  It is much like tests/periph_gpio
+but for Arduino pin numbers.
+
+In this test, pins are specified by integer pin numbers.
+It uses a single integer to indicate the arduino digital pin.
+
+At this point the Analog pins are not supported.
+
+NOTE: make sure the values you use exist on your platform! The
```

Comment:
mhm, there is a check if the number exceeds the number of defined pins, should be safe, or what do you refer to here?

Reply:
This is as per tests/periph_gpio but you're right the Arduino layer insulates against incorrect pin operation, will remove.